pkgs/development/tools/misc/stm32cubemx/default.nix

```diff
@@ -12,36 +12,37 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "stm32cubemx";
+  iconame = "STM32CubeMX";
```

Comment:
Please remove this variable. I don't think it is really required.

Reply:
Unfortunately, it is. With this new version, the filename and format of the desktop icon was changed. See lines 36, 42 and 45

Reply:
We should hardcode it there. Right now it is also exportet as an env. Alternative would be to move it into a  let in.

Reply:
Ah, good point! I moved it to the let in :)